swap/src/bin/swap.rs

```diff
@@ -22,6 +22,7 @@ use std::time::Duration;
 use structopt::StructOpt;
 use swap::bitcoin::{Amount, TxLock};
 use swap::cli::command::{AliceMultiaddress, Arguments, Command, Data, MoneroParams};
+use swap::cli::trace::init_tracing;
```

Comment:
Nit: Can we name the module `tracing` and the function `init` and call it as `tracing::init`?

Reply:
If you do that, would be good to adjust for the `asb` as well!